### ISSUE TITLE
MicMute not stating on boot

### ISSUE THREAD

**Issue by Monsix** [SaifAqqad/AHK_MicMute#49]
Hi,
Each time I boot I do not see the program running when booting even if I selected it to do so.

![imagen](https://user-images.githubusercontent.com/26342993/146026277-2e09f023-3e96-40d1-b88e-9a7aeda1325c.png)

I have to manually click on the icon in order to start it.
Not a huge problem, but it's annoying xD

Not sure if it's a bug or if I have to do something else.
Hope you can tell me anything about it.
Thanks

BTW, great program ;)

**Comment by SaifAqqad** [comment]
Can you open task scheduler and check if there is a task named `MicMute` like in the picture below:
![image](https://user-images.githubusercontent.com/47293197/146061261-31c85d2e-521e-4249-a625-0b38a1ab22db.png)

**Comment by Monsix** [comment]
Hi,

Yes, I checked Task Scheduler and I think there was an error with a previous installation from a similar program.
I saw a task named MicMute but the Location was pointing to another folder and another .exe file. ( I didn't screenshot that :_(  )
Now I have deleted that and reinstalled yours and it appears and works perfectly.

![image](https://user-images.githubusercontent.com/26342993/146133896-b52d44c5-b638-4c6a-b19a-fb3456d47973.png)

First it appeared "MicSwitch" when I reinstalled yours, and after rebooting it appeared "MicMute" and it loads in the startup perfectly.

I think everything is fine, thanks :)
